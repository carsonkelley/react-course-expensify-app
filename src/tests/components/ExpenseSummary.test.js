import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary';
import expenses from '../fixtures/expenses';
import { filters } from '../fixtures/filters'; 


test('should render ExpenseSummary with single expense correctly', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={235}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpenseSummary with multiple expenses correctly', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={23} expensesTotal={3423424}  />);
    expect(wrapper).toMatchSnapshot();
});