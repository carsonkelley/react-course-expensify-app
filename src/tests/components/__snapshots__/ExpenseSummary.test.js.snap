// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseSummary with multiple expenses correctly 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totaling 
    $34,234.24
  </h1>
</div>
`;

exports[`should render ExpenseSummary with single expense correctly 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $2.35
  </h1>
</div>
`;
